db/seeds.rb

```diff
@@ -7,6 +7,14 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 #
 
+unless User.where(email: "darrentburgess@gmail.com").first
+  User.create(email: "darrentburgess@gmail.com", password: "password", full_name: "Darren Burgess")
+end
+
+unless User.where(email: "steve@jones.com").first
```

Comment:
finding a user using `where(...).first` as show in videos is actually old way that was valid in Rails 3, but in Rails 4+ `first` also orders the records, and it does 1 more extra step compared to `find_by` that uses `where(...).take` under the hood and is a bit more effective.

We recommend to use `find_by` to search for a single record rather than `where` + `first`